src/consent-manager-builder/index.tsx

```diff
@@ -280,7 +280,13 @@ export default class ConsentManagerBuilder extends Component<Props, State> {
         shouldReload
       })
 
-      return { ...prevState, destinationPreferences, preferences, newDestinations }
+      return {
+        ...prevState,
+        destinationPreferences,
+        preferences,
+        newDestinations,
+        isConsentRequired: false
```

Comment:
can you please explain what is the reason to set `isConsentRequired` to false?

Reply:
We want the consent banner modal to disappear after the user clicks on a consent. The BCapp PR enable the update consent banner to after the consent.